cli/azd/test/functional/cli_test.go

```diff
@@ -328,7 +328,7 @@ func Test_CLI_InfraCreateAndDeleteUpperCase(t *testing.T) {
 	require.NoError(t, err)
 
 	// test 'infra create' alias
-	_, err = cli.RunCommandWithStdIn(ctx, stdinForProvision(), "infra", "create", "--output", "json")
+	_, err = cli.RunCommandWithStdIn(ctx, stdinForProvision(), "provision", "--output", "json")
```

Comment:
If we're updating this, it may be worth updating the comment too.

Reply:
missed the comment. Looks like we want coverage for the alias until it is removed